### DISCUSSION TITLE
Web page request is a little slow, whether to consider using golang to reconstruct

### DISCUSSION THREAD

**Discussion by cubxxw** [langfuse/langfuse#2653] [Ideas]
### Describe the feature or potential improvement

FYI: local deploy langfuse, web page request is a little slow, whether to consider using golang to reconstruct？

**Comment by marcklingen** [comment]
Can you share how you run Langfuse and what is slow exactly?

**Reply by cubxxw** [reply]
Click on the reaction of each woman, and the delay when the data is large

Like when I was sifting through the scores

**Reply by marcklingen** [reply]
Most of the latency is driven by database reads/writes. This will improve a lot in v3: https://github.com/orgs/langfuse/discussions/1902
